specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2021-01-01/bms.json

```diff
@@ -0,0 +1,10157 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-01-01",
+    "title": "RecoveryServicesBackupClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": false
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/backupconfig/vaultconfig": {
```

Comment:
@pilor  backupconfig doesn't seem to be an official type in the manifest, is this ok?

Reply:
child proxy types don't necessarily have to be in the manifest if they use the same endpoints as the parent